### PR TITLE
kelly/payment-methods

### PR DESCRIPTION
**Pull request by kelly-binary**

### CODE REVIEW

File: src/templates/cashier/payment_methods.html.tt [hunk 1]
```diff
@@ -1,64 +1,202 @@
-<div class="fx-section">
-    <div class="fx-payment-methods container">
+[% methods=
+    [
+        {   id=l('visa') currencies=[l('USD'), l('GBP'), l('EUR'), l('AUD')]
+            deposit   ={ min_max=l('10 - 10,000' ) process_time=l('Instant') }
```
Comment (negarn) [inline src/templates/cashier/payment_methods.html.tt]: do we want to translate these numbers? (And currencies for that matter)

Comment (kelly-binary) [reply src/templates/cashier/payment_methods.html.tt]: i feel like we don't need to @negarn  @binary-hamedanchi

File: src/templates/cashier/payment_methods.html.tt [hunk 2]
```diff
@@ -1,64 +1,202 @@
...
-        [% END %]
+                <div class="tab-content col-xs-8 col-sm-9 col-md-12">
+                    [% BLOCK tab_content %]
+                        [% FOREACH method IN methods %]
+                        <div id=[% method.id %] class="row">
+                            <div class="col-sm-12 col-md-3">
+                                <p>[% l('Currencies') %]</p>
+                                <h4>[% method.currencies.join(', ') %]</h4>
+                                <p>[% l('Processing time*') %]</p>
+                                <h4>[% l('Deposit: ') %][% method.deposit.process_time %]
```
Comment (mohammad-deriv) [inline src/templates/cashier/payment_methods.html.tt]: Better to prevent translators from accidentally removing necessary space at the end.

Comment (kelly-binary) [reply src/templates/cashier/payment_methods.html.tt]: Just remove the space? e.g: `l('Deposit:')`

File: src/javascript/pages/cashier/payment_methods.js [hunk 3]
```diff
@@ -4,24 +4,145 @@ const Client         = require('../../common/client');
...
     const load = () => {
         ChampionSocket.wait('authorize').then(() => {
-            const container = $('.fx-payment-methods');
+            $container = $('.scrollable-tabs');
+            $previousButton = $('.previous-button');
+            $nextButton = $('.next-button');
+            isVertical = $(window).innerWidth() < 767;
+            currentFirstTab = 1;
+
+            const container = $('#payment_methods');
```
Comment (mohammad-deriv) [inline src/javascript/pages/cashier/payment_methods.js]: Having two variables with the same name `$container` and `container` is ambiguous, also the second one is a jQuery object as well, so better to start with `$` for consistency.

File: src/javascript/pages/cashier/payment_methods.js [hunk 4]
```diff
@@ -4,24 +4,145 @@ const Client         = require('../../common/client');
...
-                container.find('#btn-deposit, #btn-withdraw').removeClass(hidden_class);
-                ChampionSocket.send({ cashier_password: 1 }).then((response) => {
-                    if (!response.error && response.cashier_password === 1) {
-                        container.find('#btn-deposit, #btn-withdraw').addClass('button-disabled');
-                    }
-                });
+                container.find('#btn-signup').removeClass('invisible');
+            } else {
+                container.find('#btn-cashier').removeClass('invisible');
             }
```
Comment (mohammad-deriv) [inline src/javascript/pages/cashier/payment_methods.js]: What about:
```js
container.find(Client.is_logged_in() ? '#btn-cashier' : '#btn-signup').removeClass('invisible');
```

File: src/templates/cashier/payment_methods.html.tt (hunk 2 above)
Comment (mohammad-deriv) [reply src/templates/cashier/payment_methods.html.tt]: And put the space outside, please.

**Review by mohammad-deriv — APPROVED** [review]

**Pull request merged**